### PR TITLE
EL-884: Add help page for public beta

### PR DESCRIPTION
**Pull request by patrick-laa**
[Jira ticket](https://dsdmoj.atlassian.net/browse/EL-884)

## What changed and why

Add a help page
Put link to it behind a feature flag

## Checklist

Before you ask people to review this PR:

- Tests and rubocop should be passing
- Branch is generally up to date with main Github - definitely no conflicts
- No unnecessary whitespace changes. These make diffs harder to read and conflicts more likely.
- PR description says *what* changed and *why*, with a link to the JIRA story.
- Diff has been checked for unexpected changes being included.
- Commit messages say why the change was made.

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -2174,4 +2174,18 @@ en:
     subheading_2: How to share results
     subsection_2: To share the results of a check, save the result declaration as a PDF. You can do this on the results page at the end of the check.
     cta: Start a new eligibility check
+  helps:
+    show:
+      title: Help
+      get_help: Get help to use this service
+      contact: "Contact our customer services team if you need help doing things online, or want support with legal aid administration or training:"
+      telephone: 0300 200 2020
```
Comment (MazOneTwoOne) [inline config/locales/en.yml:2182]: Shouldn't this be:

`telephone: "telephone: 0300 200 2020"`

File: config/locales/en.yml [hunk 2]
```diff
@@ -2174,4 +2174,18 @@ en:
...
+      webchat_html: on <a href="https://legalaidlearning.justice.gov.uk/totara/dashboard/index.php?id=34" target="_blank" rel="noreferrer noopener">our training website</a>
```
Comment (MazOneTwoOne) [inline config/locales/en.yml:2183]: Shouldn't this be:

`webchat_html: webchat&#58; on <a href="https://legalaidlearning.justice.gov.uk/totara/dashboard/index.php?id=34" target="_blank" rel="noreferrer noopener">our training website</a>`

**Review by MazOneTwoOne — CHANGES_REQUESTED** [review]
Just some comments so it matches the design in the ticket.

File: config/locales/en.yml (hunk 1 above)
Comment (patrick-laa) [reply config/locales/en.yml:2182]: You're absolutely right on both counts: I confused myself by copying in the text and then seeing that it already looked like YAML. Will fix.

**Review by MazOneTwoOne — APPROVED** [review]

**Pull request merged**